### DISCUSSION TITLE
Organization hostRules for helm

### DISCUSSION THREAD

**Discussion by jcogilvie** [renovatebot/renovate#25866] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

The [docs](https://docs.renovatebot.com/getting-started/private-packages/#helm) say to add a hostRule to use a private helm repo.  However, in the settings UI for my org in mend.io, the dropdown list under "Host rules" does not show helm as an option for `hostType`.  

How can I set up an org-wide authentication to my private helm repo?

**Comment by rarkins** [comment]
Can you please show a screenshot of what you see? This is part of the Mend app - not Renovate directly - and I wasn't aware it's public yet.

Cc @vital-ws @justo-mend

**Reply by jcogilvie** [reply]
Here's what I have:

<img width="938" alt="image" src="https://github.com/renovatebot/renovate/assets/679297/aaf86a57-ed44-4b95-92dd-caf26b08b97c">

**Reply by rarkins** [reply]
Thanks. That config is not meant to be released yet and not fully functional, so unfortunately my recommendation is that you configure hostRules the usual way (in presets or a repo config) and use the `encrypted` capability

**Reply by jcogilvie** [reply]
So I need a `whitesource-config` repo as indicated [here](https://docs.mend.io/bundle/integrations/page/global_repo_configuration.html)?  The renovate docs, by contrast, suggest putting shared configs in `renovate-config`.  

Forgive my ignorance as I'm just now onboarding.  

A point of feedback:  the renovate documentation on how to configure these does not seem to be complete.  It mentions using `hostRules` but not where one might put global ones beside mention of "a config file."  It took some googling to find the mend docs, which finally suggested the strongly-named repo.

**Reply by rarkins** [reply]
The mend docs are the wrong docs, do not trust your googling

**Reply by rarkins** [reply]
Here is an example from the Renovate docs: https://docs.renovatebot.com/getting-started/private-packages/#encrypting-secrets

**Reply by jcogilvie** [reply]
Thank you.  I can see how to write the contents of a config file, but I do not know where to put it so that it is global for my org.  Is there a need for a strongly named repo?

**Reply by rarkins** [reply]
Yes
